docs/manual/cli.md

```diff
@@ -31,8 +37,8 @@ The reply of the command is "7". Since Redis replies are typed (strings, arrays,
     $ cat /tmp/output.txt
     8
 
-Notice that `(integer)` was omitted from the output since `redis-cli` detected
-the output was no longer written to the terminal. You can force raw output
+Note that `(integer)` is omitted from the output because `redis-cli` detects
+the output is no longer written to the terminal. You can force raw output
 even on the terminal with the `--raw` option:
```

Comment:
Feel free to concat lines and break after a period if you're editing stuff (i.e. no need to break at 80 chars)

Reply:
thank you